examples/attention/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# The atom_types.h file is written to the build directory
+
+# PLNDynamicExpSetupModule
+INCLUDE_DIRECTORIES(${CMAKE_BINARY_DIR})
+
+IF (TBB_FOUND)
```

Comment:
I'll try to get rid of it. I think I really need to become more sceptical of code I modify. Instead of leaving stuff like it is when I am not quite sure what purpose it serves.